build.yaml

```diff
@@ -0,0 +1,11 @@
+targets:
```

Comment:
I don't think any of this config is needed, all of the tests are VM tests and we don't depend on `build_web_compilers` in this package